scss/variables/typography.js

```diff
@@ -52,7 +52,65 @@ chassis.typography = {
 	sans: {
 		name: "Font Family - Sans",
 		value: "Helvetica, Arial, sans-serif"
-	}
+	},
+	"blockquote-color": {
+		name: "blockquote color",
+		value: "#eee"
+	},
+  "blockquote": {
+    name: "Blockquote",
+    value: {
+      "margin": "0",
+      "padding-left": "1em",
+      "position": "relative",
+      "border-left": "4px solid ",
+      "font-style": "italic"
+    }
+  },
+	"code-background": {
+		name: "Code Background Color",
+		value: "#F2F2F2"
+	},
+  "code": {
+    name: "Code",
+    value: {
+      "background": "$code-background",
```

Comment:
Redundant?? Should use only one? either code-background or inside a map. And since you seem to use maps everywhere might as well stick to maps here too.